docs/mobile/authoring.md

```diff
@@ -0,0 +1,128 @@
+# Authoring
+
+Writing a DataCamp for Mobile course is a unique experience. The upside of the
+mobile platform is that you can reach a much broader audience because the
+convenience is higher and the barrier to entry is lower. The challenge is that
+there are many more constraints to work within. Primary among these are the
+restricted screen real estate and the lack of an interactive interpreter. These
+constraints require you to think carefully about the way you introduce concepts
+and the examples that you use.
+
+## Getting started
+
+Create and checkout a feature branch to write your course in. For example:
```

Comment:
Clone the repo first?

Reply:
This is currently in the Development Process page, which is the intended starting point. The recommended documentation order is the order on the sidebar. I'll add a note at the beginning that you should go through that page first.